packages/js/api-client/src/auth.ts

```diff
@@ -136,6 +136,12 @@ export class OpenverseAuthMiddleware implements OpenverseMiddleware {
             client_id: this.credentials.clientId,
             client_secret: this.credentials.clientSecret,
           },
+          bodySerializer(body) {
+            return new URLSearchParams(body).toString()
+          },
+          headers: {
+            Accept: "x-www-form-encoded",
```

Comment:
```suggestion
           "Content-Type": "x-www-form-encoded",
```
Sorry, I've mixed up the headers in the issue. The correct one will be content type, to encode the body of the **request** (and not the response)

Reply:
Skipped this myself, thanks for spotting. Updated now.